### ISSUE TITLE
Docker Installation -> npm not found

### ISSUE THREAD

**Issue by letmp** [RestyaPlatform/board#2373]
I am trying to build the board with docker but it fails:
```
Step 10/49 : RUN npm install
 ---> Running in f68ccb205e1c
/bin/sh: 1: npm: not found
```

That is very strange since step 6 and 7 returned no errors:
```
Step 6/49 : RUN curl -sL https://deb.nodesource.com/setup_10.x | bash -
 ---> Running in 84af281b9a5a
Removing intermediate container 84af281b9a5a
 ---> 969d848a0026
Step 7/49 : RUN apt-get install -y nodejs
 ---> Running in 6944cac7c9c2
```

I expect that npm was successfully installed with RUN apt-get install -y nodejs
Any ideas why npm could not be found?

Thanks!

**Comment by sridhar391** [comment]
@letmp  Sorry, we aren't familiar with docker @BlackDark Can you please try to solve this issue

**Comment by BlackDark** [comment]
Old issue.
@letmp 
```
Removing intermediate container 84af281b9a5a
```
Somehow the layer got removed with the `npm` installed. Probably fixed by docker now.
At least it will be fixed with #4013 because there is a new setup.

@sridhar391 you can assign me and close this issue.
Nothing to do here.

**Comment by rrjanbiah** [comment]
Cl
